types/mac.go

```diff
@@ -14,3 +16,24 @@ type MACAddress [EtherAddrLen]uint8
 func (mac MACAddress) String() string {
 	return fmt.Sprintf("%02x:%02x:%02x:%02x:%02x:%02x", mac[0], mac[1], mac[2], mac[3], mac[4], mac[5])
 }
+
+func StringToMACAddress(str string) (MACAddress, error) {
```

Comment:
Could you please all these functionality to Wiki

Reply:
Ok